<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="videoCature">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>611</width>
      <height>631</height>
     </rect>
    </property>
    <property name="text">
     <string>видео</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="automaticMode">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Автоматический режим</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchNumber">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>350</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть поиск номеров</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="carParking">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>381</width>
      <height>301</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Номер</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ФИО</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата заезда</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="openBarrier">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>660</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть шлагбаум</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBarrier">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>720</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть шлагбаум</string>
    </property>
   </widget>
   <widget class="QLabel" name="barrierLable">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>660</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>close_barrier.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="editorTexting">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>440</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveText">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>720</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить текст</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletText">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>720</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить текст</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
